client/flutter/lib/pages/onboarding/notifications_page.dart

```diff
@@ -23,90 +23,111 @@ class _NotificationsPageState extends State<NotificationsPage> {
+              Expanded(
+                flex: 7,
+                child: SingleChildScrollView(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: <Widget>[
+                      Container(
+                        alignment: Alignment.center,
+                        padding: EdgeInsets.only(
+                          left: 36.0,
+                          bottom: 18.0,
+                          top: 10,
                         ),
+                        child: SvgPicture.asset(
+                            'assets/svg/undraw-onboarding-notifications.svg'),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.only(
+                            left: 36.0, right: 20.0, bottom: 18.0),
                         child: ThemedText(
                           S
                               .of(context)
-                              .notificationsPagePermissionRequestPageButton,
-                          variant: TypographyVariant.button,
+                              .notificationsPagePermissionRequestPageTitle,
+                          variant: TypographyVariant.h2,
                           style: TextStyle(
-                            color: CupertinoColors.white,
+                            color: Constants.accentNavyColor,
+                            fontWeight: FontWeight.w800,
                           ),
-                          textAlign: TextAlign.center,
                         ),
                       ),
-                    ),
-                    CupertinoButton(
-                      onPressed: () async {
-                        await _skipNotifications();
-                      },
-                      child: ThemedText(
-                        S.of(context).commonPermissionRequestPageButtonSkip,
-                        variant: TypographyVariant.button,
-                        style: TextStyle(
-                          color:
-                              Constants.neutralTextLightColor.withOpacity(0.5),
+                      Padding(
+                        padding: EdgeInsets.only(
+                            left: 36.0, right: 54.0, bottom: 12.0),
+                        child: ThemedText(
+                          S
+                              .of(context)
+                              .notificationsPagePermissionRequestPageDescription,
+                          variant: TypographyVariant.body,
+                          style: TextStyle(
+                            color: Constants.neutral1Color,
+                          ),
                         ),
-                      ),
-                    )
-                  ],
+                      )
+                    ],
+                  ),
+                ),
+              ),
+              Expanded(
+                flex: 3,
+                child: SingleChildScrollView(
```

Comment:
This view has two isolated scroll views, which could cause potential usage problems for a user who doesn't want to scroll through the buttons. Possibly one or both should be a Column.

@davidkaneda what is your thought?

Reply:
my concern was the size of the images on older phones / small screens, but in this last commit I tried to control only with the flex widgets